portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -60,9 +63,19 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String message = getParameter(request, "text-input", "");
+
+    // Sentiment Analysis
+    Document doc = Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
+    Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+    float score = sentiment.getScore();
+    System.out.println("Score: " + score);
+    languageService.close();
+
     long timestamp = System.currentTimeMillis();
 
     Entity messageEntity = new Entity("Message");
+    messageEntity.setProperty("score", score);
```

Comment:
ok, looking at this PR immediately after the first one, I find myself a little confused, as this seems to just be putting back the printing (and datastore integration) that was removed from the first PR. What is the purpose of this PR (or perhaps it should just be a discussion of how you're deciding to split your PRs)?

Reply:
Ah yes. So this is what I think happened. I somehow deleted the print statement in the previous PR. That was supposed to be the only purpose for the previous PR. The whole purpose for this PR is simply adding it to the datastore. I decided to do it this way (in 2 separate PRs) as it mentioned it in the walkthrough.